README.md

```diff
@@ -10,19 +10,18 @@ This codeception module is based on [oqq/codeception-email-mailhog](https://gith
 brings nearly the same functionality for Mailpit as the mentioned modules did for MailHog.
 
 ## Installation
-Through composer, require the package:
+With Composer:
 ```shell
-composer req koehnlein/codeception-email-mailpit --dev
+composer require koehnlein/codeception-email-mailpit --dev
```

Comment:
I don't see a need to change this line. Why would you do it?

Reply:
Just to explain what `req` stands for, cause some newbies might not understand it :-)